chcemvediet/templates/main/homepage/main.html

```diff
@@ -15,7 +15,7 @@
 {% block content %}
   <div class="chv-scheme-inv-blue">
     <div class="container text-center">
-      {% include "main/homepage/search.html" %}
+      {% include "main/snippets/search.html" with q="" search_text=True %}
```

Comment:
Treba `q=""` neda sa pouzit vo vnutri includnuteho templatu nieco ako `q|default_if_none:""`? Pripadne to oifovat?

https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#default-if-none

Reply:
Vlastne nebude treba parameter `q` includovat vobec.  Ak ho tam nedame tak potom bude `q` None, a to je vlastne to iste ako by keby bolo `""`, v nasom pripade.